middleman-core/features/asset_hash.feature

```diff
@@ -112,7 +112,7 @@ Feature: Assets get file hashes appended to them and references to them are upda
     Given a fixture app "asset-hash-host-app"
     And a file named "config.rb" with:
       """
-      set :sass_source_maps, false
+      config[:sass][:source_map] = false
```

Comment:
This is a user-facing API change. Can we avoid this?

Reply:
Done.